modules/vardictjava/meta.yml

```diff
@@ -0,0 +1,57 @@
+name: "vardictjava"
+
+description: The Java port of the VarDict variant caller
+keywords:
+  - variant calling
+  - VarDict
+  - AstraZeneca
+tools:
+  - "vardictjava":
+      description: "Java port of the VarDict variant discovery program"
+      homepage: "https://github.com/AstraZeneca-NGS/VarDictJava"
+      documentation: "https://github.com/AstraZeneca-NGS/VarDictJava"
+      tool_dev_url: "https://github.com/AstraZeneca-NGS/VarDictJava"
+      doi: "10.1093/nar/gkw227 "
+      licence: "['MIT']"
+
+input:
+  - meta:
+      type: map
+      description: |
+        Groovy Map containing sample information
+        e.g. [ id:'test', single_end:false ]
+
+  - bam:
+      type: file
+      description: BAM/SAM file
+      pattern: "*.{bam,sam}"
+
+  - reference_fasta:
```

Comment:
`fasta_fai` is missing
change to `fasta` and `fasta_fai` in accordance to naming conventions